Src/contrib/library/web/authentication/oauth/cypress/consumer/examples/box_oauth_20_jwt_api_example.e

```diff
@@ -0,0 +1,101 @@
+note
+	description: "Summary description for {BOX_OAUTH_20_JWT_API_EXAMPLE}."
+	date: "$Date: 2015-06-18 16:46:10 -0300 (ju. 18 de jun. de 2015) $"
+	revision: "$Revision: 97507 $"
+	EIS: "name=OAuth 2.0 for Server to Server Applications", "src=https://developers.google.com/identity/protocols/OAuth2ServiceAccount", "protocol=uri"
+
+
+class
+	GOOGLE_OAUTH_20_JWT_API_EXAMPLE
+
+create
+	make
+
+feature -- Access
+
+	make
+		local
+			oauth_jwt: OAUTH_20_JWT_API
+			config: OAUTH_JWT_CONFIG
+			api_service: OAUTH_20_JWT_SERVICE
```

Comment:
it is enough to declare as OAUTH_SERVICE_I 
and then later ... remove the "{OAUTH_20_JWT_SERVICE} / ..."